src/lib/utils/music/ButtonHandler.ts

```diff
@@ -106,27 +110,28 @@ export async function embedButtons(
             }
             if (i.customId === 'stop') {
               await i.update('Leaving');
-              await deletePlayerEmbed(player.queue);
-              player?.disconnect();
-              client.music.destroyPlayer(player.guildId);
+              await deletePlayerEmbed(queue);
+              await queue.leave();
               clearTimeout(timer);
             }
             if (i.customId === 'next') {
               await i.update('Skipping');
-              player.queue.next();
+              await queue.next({ skipped: true });
               clearTimeout(timer);
             }
```

Comment:
Missing the Embed call when the Next button is pressed

what I found that worked for me
```
            if (i.customId === 'next') {
              await i.update('Skipping');
              await queue.next({ skipped: true });
              const NowPlaying = new NowPlayingEmbed(
                song,
                0,
                tracks[0].length ?? 0,
                await queue.getVolume(),
                await queue.tracks(0, -1),
                tracks.at(-1),
                paused
              );
              await embedButtons(NowPlaying.NowPlayingEmbed(), queue, song);
              clearTimeout(timer);
            }
```

Much Love
-Bacon